src/lua/fiber.c

```diff
@@ -938,6 +939,20 @@ lbox_fiber_set_system(struct lua_State *L)
 	return 0;
 }
 
+/** Turn managed shutdown for fiber. Takes the fiber as single argument. */
+static int
+lbox_fiber_set_managed_shutdown(struct lua_State *L)
+{
+	if (lua_gettop(L) != 1) {
+		diag_set(IllegalParams,
+			 "fiber.set_managed_shutdown(id): bad arguments");
+		luaT_error(L);
+	}
+	struct fiber *fiber = lbox_checkfiber(L, 1);
+	fiber_set_managed_shutdown(fiber);
+	return 0;
+}
+
```

Comment:
It'd be nice to add it in the same commit where you add `fiber_set_managed_shutdown`.